### PR TITLE
#167765378 setup sequelize

### PR DESCRIPTION
**Pull request by ugwumadu116**
#### What does this PR do?
- Sets up Sequelize for PostgreSQL
#### Description of Task to be completed?
- install sequelize-cli, sequelize, and pg-hstore 
- add .sequelizerc to configure sequelize-cli
#### How should this be manually tested?
- Install all dependencies by running `npm install`
- Create a Postgresql database and update `development` in the `./src/config/config.json` file with your database credentials
- Run `npm run migrate`
#### Any background context you want to provide?
- Install `sequelize-cli` globally by running `npm i -g sequelize-cli`. This will give you access to the sequelize command
#### What are the relevant pivotal tracker stories?
#167765378
#### Implementation Plan
- https://docs.google.com/document/d/19eClN5h0qNu9wjhHe4IRYpM9z_zXTlsp4MLX_YQIlzI/edit?usp=sharing

### CODE REVIEW

**Review by debelistic — APPROVED** [review]
Nice work

File: package.json [hunk 1]
```diff
@@ -20,11 +21,14 @@
     "jsonwebtoken": "^8.3.0",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
-    "mongoose": "^5.2.2",
-    "mongoose-unique-validator": "^2.0.1",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
+    "path": "^0.12.7",
```
Comment (uggoprince) [inline package.json:33]: You don't need to install `path`. It comes bundled with `npm`.

**Review by uggoprince — DISMISSED** [review]
@ugwumadu116  you're suppose to run `sequelize init` inside the `database` directory by the using this path `./src/database/...` in `.sequelizerc`

File: package.json (hunk 1 above)
Comment (ugwumadu116) [reply package.json:33]: Ok that's cool. I did that in case of an upgrade.

**Pull request closed without merging**